simulator/cube.py

```diff
@@ -2,16 +2,18 @@
 import display
 import event
 import obj_canvas
+from config import *
+import cube_logger
 
 class Cube(object):
   """ Represents a single cube. """
 
   class Colors(object):
     """ Represents cube colors. """
 
-    RED = colors.CUBE_RED
-    BLUE = colors.CUBE_BLUE
-    GOLD = colors.CUBE_GOLD
+    RED = CUBE_RED
```

Comment:
Instead of aliasing these constants, I would get rid of the Colors class entirely and use the definitions from the config file directly.